core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectsResource.java

```diff
@@ -136,6 +137,59 @@ public void lookUpSchemaUnderSubject(
     asyncResponse.resume(matchingSchema);
   }
 
+  @GET
+  @DocumentedName("getLatestWithMetadata")
+  @Path("/{subject}/metadata")
+  @Operation(summary = "Retrieve the latest version with the given metadata.",
+      description = "Retrieve the latest version with the given metadata.",
+      responses = {
+          @ApiResponse(responseCode = "200", description = "The schema", content = @Content(schema =
+          @io.swagger.v3.oas.annotations.media.Schema(implementation = Schema.class))),
+          @ApiResponse(responseCode = "404", description = "Error code 40401 -- Subject not found\n"
+              + "Error code 40403 -- Schema not found"),
+          @ApiResponse(responseCode = "500", description = "Internal server error")
+      })
+  @PerformanceMetric("subjects.get-latest-with-metadata")
+  public void getLatestWithMetadata(
+      final @Suspended AsyncResponse asyncResponse,
+      @Parameter(description = "Subject under which the schema will be registered", required = true)
+      @PathParam("subject") String subject,
+      @Parameter(description = "The metadata key")
```

Comment:
I assume these metadata Key-value pairs are the properties that we used to match a SchemaMetadata, is that correct?

Reply:
Yes